src/test/java/gov/cabinetofice/gapuserservice/web/LoginControllerV2Test.java

```diff
@@ -112,8 +116,15 @@ void shouldRedirectToLoginPage_IfTokenIsNull_AndMigrationJourneyDisabled() {
 
             final RedirectView methodResponse = loginController.login(redirectUrl, request, response);
 
-            final Cookie stateCookie = WebUtil.buildSecureCookie(loginController.getSTATE_COOKIE(), "eyJyZWRpcmVjdFVybCI6Imh0dHBzOlwvXC93d3cuZmluZC1nb3Zlcm5tZW50LWdyYW50cy5zZXJ2aWNlLmdvdi51a1wvIiwic3RhdGUiOiJzdGF0ZSJ9", 3600);
-            verify(response).addCookie(stateCookie);
+            ArgumentCaptor<Cookie> argument = ArgumentCaptor.forClass(Cookie.class);
+            verify(response).addCookie(argument.capture());
+            Cookie stateCookie = argument.getValue();
+
+            assertThat(stateCookie.getName()).isEqualTo(loginController.getSTATE_COOKIE());
+            assertThat(stateCookie.getValue()).isEqualTo("eyJyZWRpcmVjdFVybCI6Imh0dHBzOi8vd3d3LmZpbmQtZ292ZXJubWVudC1ncmFudHMuc2VydmljZS5nb3YudWsvIiwic3RhdGUiOiJzdGF0ZSJ9");
+            assertThat(stateCookie.isHttpOnly()).isEqualTo("true");
+            assertThat(stateCookie.getSecure()).isEqualTo("true");
+            assertThat(stateCookie.getMaxAge()).isEqualTo("3600");
```

Comment:
this refactor isn't necessary to get the test to pass, but it will log which bit of the cookie is is incorrect if that fails again in future